### PR TITLE
[Debug + refactor] 

### PR DESCRIPTION
**Pull request by ziyunp**
1. only log successful trx 
2. remove printouts
3. rename ExecutionTrace to CallTrace

### CODE REVIEW

File: libethploit/EVMAnalyser.cpp [hunk 1]
```diff
@@ -61,15 +58,20 @@ bool EVMAnalyser::isEthploitModeEnabled() {
...
                                    std::string _transactionHash, 
                                    dev::u256 senderBalance, 
                                    dev::u256 receiverBalance,
                                    int64_t blockNumber) {
     if (senderBalance != -1 && receiverBalance != -1) {
         if (transactionHash != _transactionHash) {// New transaction
+            badTransaction = false;
             logger->setTransactionInfo(account, _transactionHash, blockNumber);
-
+            badTransaction = false;
```
Comment (Au-ChiKin) [inline libethploit/EVMAnalyser.cpp]: Why there is a duplicate line of setting badTransaction false?

File: libethploit/ExecutionTrace.cpp [hunk 2]
```diff
@@ -5,7 +5,7 @@ namespace dev
     namespace eth 
     {
 
-        ExecutionTrace::ExecutionTrace(Instruction _instruction, 
+        CallTrace::CallTrace(Instruction _instruction,
```
Comment (Au-ChiKin) [inline libethploit/ExecutionTrace.cpp:8]: Maybe change the file name as well?

File: libethploit/ExecutionTrace.h [hunk 3]
```diff
@@ -17,16 +17,16 @@ namespace eth
 
 
 
-struct ExecutionTrace 
+struct CallTrace
```
Comment (Au-ChiKin) [inline libethploit/ExecutionTrace.h:20]: Change this file name as well?

**Review by Au-ChiKin — APPROVED** [review]
They should be fine.

File: libethploit/EVMAnalyser.cpp (hunk 1 above)
Comment (ziyunp) [reply libethploit/EVMAnalyser.cpp]: This is to reset for the next trx if the prev is set to true. If that's the duplication u mean

**Pull request merged**